tests/ffi/common/prepare.sh

```diff
@@ -67,3 +67,9 @@ run_container_in_qm() {
       tail -f /dev/null\""
    exec_cmd "${run_ctr_in_qm}"
 }
+
+init_ffi() {
```

Comment:
In a second patch, please add a comment into this function, like docstring explaning what this it does, so people can read the code and understand what we are trying to do. :)

Reply:
Done.